File: super-impactor source/C#/AppConst.cs
using log4net;
using System;
using System.Collections.Generic;
using System.Data.SQLite;

namespace SuperImpactor
{
	public class AppConst
	{
		public const string API_SENDLOG = "http://superimpactor.net/api/send_log.php";

		public const string DB = "\\iaidb.sqlite";

		public const string IMAGE = "\\res\\";

		public const string IMOBILEDEVICE = "\\libimobiledevice\\";

		public const string OPENSSL = "\\openssl\\bin\\";

		public const string ZIP7 = "\\7zip\\";

		public const string CERTSTORE = "\\certs\\";

		public const string DOWNLOAD = "\\download\\";

		public const string LOCALTMP = "\\tmp\\";

		public static string m_localTmp;

		public static SQLiteConnection m_dbConnection;

		public static Dictionary<int, string> m_lstCydiaRepos;

		public static string m_rootPath;

		public static string m_randomKey;

		public static ILog logger;

		public static frmSuperImpactor mainForm;

		public AppConst()
		{
		}
	}
}

File: super-impactor source/VB/AppConst.vb
Imports log4net
Imports System
Imports System.Collections.Generic
Imports System.Data.SQLite

Namespace SuperImpactor
	Public Class AppConst
		Public Const API_SENDLOG As String = "http://superimpactor.net/api/send_log.php"

		Public Const DB As String = "\iaidb.sqlite"

		Public Const IMAGE As String = "\res\"

		Public Const IMOBILEDEVICE As String = "\libimobiledevice\"

		Public Const OPENSSL As String = "\openssl\bin\"

		Public Const ZIP7 As String = "\7zip\"

		Public Const CERTSTORE As String = "\certs\"

		Public Const DOWNLOAD As String = "\download\"

		Public Const LOCALTMP As String = "\tmp\"

		Public Shared m_localTmp As String

		Public Shared m_dbConnection As SQLiteConnection

		Public Shared m_lstCydiaRepos As Dictionary(Of Integer, String)

		Public Shared m_rootPath As String

		Public Shared m_randomKey As String

		Public Shared logger As ILog

		Public Shared mainForm As frmSuperImpactor

		Public Sub New()
			MyBase.New()
		End Sub
	End Class
End Namespace